truss/cli/cli.py

```diff
@@ -64,6 +64,23 @@
     ]
 }
 
+HELLO_PY = """import random
+import truss_chains as chains
+
+class RandInt(chains.ChainletBase):
```

Comment:
Couple things to add: 
* I think that it would be good to add some comments explaining what's going on here
* Could we link to the docs at the top as well?

Reply:
done and done!